src/WhatsAppCloudApi.php

```diff
@@ -289,13 +315,23 @@ public function sendList(string $to, string $header, string $body, string $foote
         return $this->client->sendMessage($request);
     }
 
-    public function sendButton(string $to, string $body, ButtonAction $action, ?string $header = null, ?string $footer = null): Response
+    /**
+     * Sends a button reply message.
+     *
+     * @param  string       $to             WhatsApp ID or phone number for the person you want to send a message to.
+     * @param  string       $body           The body of the message.
+     * @param  ButtonAction $action    The button action.
+     * @param  string|null  $footer      The footer text.
+     * @return Response
+     *
+     * @throws Response\ResponseException
+     */
+    public function sendButton(string $to, string $body, ButtonAction $action, ?string $footer = null): Response
```

Comment:
sendButton is a public method, if you remove the arguments from a public method you are breaking all projects that update the version of this package.

Reply:
Change Reverted